setup.py

```diff
@@ -1,34 +1,49 @@
 import re
 from glob import glob
 
+import pkg_resources
 from setuptools import find_packages, setup
 
 import versioneer
 
 
 def get_extras_require():
     results = {}
-    extra_keys = {
+    extra_key_mapping = {
         "aws_secrets": "boto",
         "azure_secrets": "azure",
         "gcp": "bigquery",
         "s3": "boto",
     }
+    sqla_keys = (
+        "athena",
+        "bigquery",
+        "dremio",
+        "mssql",
+        "mysql",
+        "postgresql",
+        "redshift",
+        "snowflake",
+        "teradata",
+    )
```

Comment:
Any way to know to keep this updated as we add more providers?

Reply:
It'll be a part of whatever instructions on adding a new dialect (being created after trino support is complete)